bin/cleanupDataOnTmp.sh

```diff
@@ -191,7 +191,8 @@ else
 			if [[ $(((dateInSecNow - dateInSecAnalysisData) / 86400)) -gt 10 ]]
```

Comment:
Die 10 dagen is nu in meerdere regels hard coded. Ik zou daar een variabele van maken, want als het later een keer aangepast moet zorden zul je anders zien dat het ergens vergeten wordt en dan wordt het verwarrend.